service/src/main/java/bio/terra/externalcreds/models/BondFenceServiceAccountEntity.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.externalcreds.models;
+
+import com.google.cloud.datastore.Key;
+import java.time.Instant;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface BondFenceServiceAccountEntity extends WithBondFenceServiceAccountEntity {
```

Comment:
This looks very similar to `FenceAccountKey` 🤔  How are they related?

Reply:
Oh it looks like `BondFenceServiceAccountEntity` is the model for the info we get from Bond and `FenceAccountKey` is just used for the ECM representation?

Reply:
Exactly! The `BondDatastoreDAO` is bond-specific, and the `BondService` handles converting Bond models to ECM ones. I didn't want to encode a lot of business logic in the DAO layer.